specification/metrics/listcost.md

```diff
@@ -0,0 +1,36 @@
+# List Cost
+
+List Cost represents the cost calculated based on [List Unit Price](#listunitprice) and the corresponding [Pricing Quantity](#pricingquantity). List cost is denominated in the [Billing Currency](#billingcurrency) and is used for calculating savings based on various rate optimization activities, by comparing it with [Effective Cost](#effectivecost).
```

Comment:
Is the following calculation true?

> List Cost = List Unit Price * Pricing Quantity

If so, could we be explicit somewhere in this document that A = B * C?  Even a change as simple as the following would do it for me:

```suggestion
List Cost represents the cost calculated by multiplying [List Unit Price](#listunitprice) and the corresponding [Pricing Quantity](#pricingquantity). List cost is denominated in the [Billing Currency](#billingcurrency) and is used for calculating savings based on various rate optimization activities, by comparing it with [Effective Cost](#effectivecost).
```

Based on this specific verbiage, I suspect this was already discussed... but given that it's vague, it leaves me as the layman reader wondering what the precise calc is.

Reply:
Its my understanding this format is consistent across all of the Price * Quantity based Cost Metrics. So if we do change and add this specifically I think it will need to be across the board.

Reply:
we'll use this as a reference for upcoming unit prices...